cmd/plural/init.go

```diff
@@ -21,7 +21,10 @@ import (
 	"github.com/pluralsh/plural/pkg/wkspace"
 )
 
-func handleInit(c *cli.Context) error {
+const DemoingErrorMsg = "You're currently running a gcp demo cluster. Spin that down at https://app.plural.sh/shell before beginning a local installation"
+
+func (p *Plural) handleInit(c *cli.Context) error {
+	p.InitPluralClient()
```

Comment:
This doesn't have any issues when there's no config created yet?

Reply:
done